compiler/compile/OMRCompilation.cpp

```diff
@@ -669,6 +670,8 @@ bool OMR::Compilation::isPotentialOSRPoint(TR::Node *node)
              self()->getSymRefTab()->getNonhelperIndex(self()->getSymRefTab()->getLastCommonNonhelperSymbol()))
             potentialOSRPoint = disableGuardedCallOSR == NULL;
          }
+      else if (node->getOpCodeValue() == TR::monent)
+         potentialOSRPoint = disableMonentOSR == NULL;
```

Comment:
potentialOSRPoint = (disableMonentOSR == NULL); 

would be slightly easier to read. Could you please make this change ?